python/graphstorm/config/argument.py

```diff
@@ -578,10 +579,17 @@ def training_method(self):
                     "em_order_gnn_first": False,
                     "inference_using_gnn": True,
                     "pl_weight": 0.5,
-                    "num_pretrain_epochs": 1
+                    "num_pretrain_epochs": 1,
+                    "lm_param_group": ["pure_lm", "node_proj_matrix"],
+                    "gnn_param_group": ["node_input_projs"]
```

Comment:
if the default config works the best, let's not expose the parameters for now. people can always tune them with GraphStorm API, instead of command line.

Reply:
All GLEM hparams can only be set in the yaml. User can't set them in command line.

Reply:
i mean GraphStorm's programming interface, instead of command line interface.